libc/stdlib_strto.c

```diff
@@ -0,0 +1,645 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ * HEADER:
+ *    - stdlib.h
+ *
+ * TESTED:
+ *    - strtod()
+ *    - strtof()
+ *    - strtold()
+ *    - strtol()
+ *    - strtoll()
+ *    - strtoul()
+ *    - strtoull()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Mateusz Bloch
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdlib.h>
+#include <float.h>
+#include <math.h>
+#include <limits.h>
+#include <errno.h>
+#include <unity_fixture.h>
+
+
+TEST_GROUP(stdlib_strto);
+
+
+TEST_SETUP(stdlib_strto)
+{
+}
+
+
+TEST_TEAR_DOWN(stdlib_strto)
+{
+}
+
+
+TEST(stdlib_strto, strtod_basic)
+{
+	const char *str[] = { "0", "-0", "-1", "1", "+1", ".1", "3.1415", "-3.1415", "0.1234567891234567",
```

Comment:
I would also prefer to define DBL_MAX and DBL_MIN and test them too. Applies to the other cases.